paper_slides/SConstruct

```diff
@@ -8,14 +8,18 @@ sys.dont_write_bytecode = True # Don't write .pyc files
 
 # Setup
 from configuration import configuration
-[mode, vers, cache_dir, PATHS] = configuration(ARGUMENTS, paper = True)
+[mode, vers, cache_dir, PATHS, pythonpath] = configuration(ARGUMENTS, paper = True)
 import gslab_scons as gs
 
 # Start log
 gs.log.start_log(mode, vers)
 
 # Define the SCons environment
-env = Environment(ENV = {'PATH': os.environ['PATH']}, 
+try:
+   pythonpath = os.environ['PYTHONPATH']
+except:
+   pythonpath = ''
```

Comment:
We're defining `pythonpath` twice in this file.

Reply:
Whoops, forgot to delete this block.